content/docs/refs-and-the-dom.md

```diff
@@ -11,6 +11,8 @@ redirect_from:
 permalink: docs/refs-and-the-dom.html
 ---
 
+Refs provide a way to access DOM nodes and React elements created by React by getting a _reference_ to the element in the render method.
```

Comment:
I think there's something awkward about this wording.

"_React elements created by React_" sounds a bit redundant and "_getting a reference to the element in the render method_" sounds potentially misleading (since refs are rarely used _in_ `render` in my experience).

Maybe something simpler? Like:

> Refs provide a way to access DOM nodes or React elements created in the `render` method.

Reply:
Refs provide a way to _reference_ (get a reference to?) DOM nodes and React elements created by React.

Reply:
It might be obvious, but at some point in the page it should probably say that "ref" is short for "reference"... 💭